doc/nrf/protocols/thread/device_types.rst

```diff
@@ -95,31 +119,35 @@ Minimal Thread Devices can be further divided into three categories:
      - No
      - No
      - No
-   * - Radio on
-     - Always
-     - Periodically
-     - Periodically
+   * - Radio
+     - Always enabled
+     - Periodically enabled
+     - Periodically enabled
    * - New messages
      - Parent immediately forwards the messages
      - The SED polls for new messages when it wakes up
-     - Parent forwards the messages during designated transmission window
+     - Parent forwards the messages during designated transmission window or when SSED explicitly polls data
    * - No new messages
      - No data transmission
      - Parent indicates no pending messages
-     - No data transmission
+     - No data transmission*
 
-MEDs are the most basic MTDs, and their radio is always on.
-They don't keep a routing table, but otherwise operate like :ref:`FEDs <thread_types_ftd>`.
+MEDs are the most basic MTDs, and their radio is always enabled.
+They don't keep a routing table and have a :ref:`more limited range of functionalities than FEDs <thread_types_ftd>`.
 
 SEDs try to limit their power consumption by sleeping most of the time, waking up periodically to poll for messages from their parent.
 After waking up, they send a data request to their parent.
-If the parent has any pending messages, it will send them to the SED.
+If the parent has any pending messages, it will send them to the SED (which can happen at any time, since Routers have their radio enabled all the time.)
 Otherwise, the parent will send a response indicating no pending messages.
 
 SSEDs operate similarly to Sleepy End Devices, but they are synchronized with their parent.
-They wake up at the same time as their parent, eliminating the need for polling for messages.
+They wake up at designated transmission windows agreed with their their parent, which eliminates the need for polling for messages.
 If the parent has messages for the SSED, it sends them during the designated transmission window.
 The SSED allows the transmission to finish if radio activity is detected during the transmission window.
+
+.. note::
+    If the length of the message exceeds the length of the transmission window, the first frame is received in the designated transmission window, but the rest is transmitted using regular Data Polls, exactly like SED.
+
 Conversely, if there is no radio activity during the specified duration of transmission window, this indicates that the parent has no messages for the SSED and the SSED returns to sleep.
 The SSED synchronization results in lower power consumption compared to an SED, primarily because the SSED doesn't need to poll for messages, keeping transmission windows short.
```

Comment:
```suggestion
The SSED synchronization usually results in lower power consumption compared to an SED, primarily because the SSED doesn't need to poll for messages, keeping transmission windows short.
```